ctapipe/io/hdf5eventsource.py

```diff
@@ -345,6 +347,15 @@ def _generate_events(self):
                 prefixes="true",
             )
             data.simulation = SimulatedEventContainer()
+            if "impact" in self.file_.root.simulation.event.telescope:
```

Comment:
Tests fail because there is no telescope group in the test file

Reply:
There should be...

Reply:
fixed